packages/edit-site/src/components/sidebar-edit-mode/style.scss

```diff
@@ -101,11 +101,11 @@
 }
 
 .edit-site-sidebar-fixed-bottom-slot {
-	position: sticky;
+	position: absolute;
```

Comment:
I had to change this to get around the overflow-x scroll in Safari. 

<img width="285" alt="Screenshot 2023-12-20 at 12 00 25 pm" src="https://github.com/WordPress/gutenberg/assets/6458278/deec996c-5047-4695-bac3-5ee73788d2a4">

The overflow is actually on 

`.interface-complementary-area.edit-site-global-styles-sidebar` 

but I decided on a more local approach to avoid side-effects.